standalone/src/DelphesPythia8Reader.h

```diff
@@ -67,6 +68,11 @@ class DelphesPythia8Reader : public DelphesInputReader {
       throw std::runtime_error("can't create Pythia instance");
     }
 
+    //load and initialize the ResonanceDecayFilterUserhook
+    // bool resonanceDecayFilter = m_pythia->settings.flag("ResonanceDecayFilter:filter"); //is this needed? if statement with it as in k4gen doesnt work
+    m_resonanceDecayFilterHook = new Pythia8::ResonanceDecayFilterHook(m_pythia->settings);
+    m_pythia->addUserHooksPtr((Pythia8::UserHooksPtr)m_resonanceDecayFilterHook);
```

Comment:
Does pythia take ownership here, or do we need to make the `m_resonanceDecayFilterHook` a `std::unique_ptr`?